Q4.js

```diff
@@ -0,0 +1,26 @@
+//In countries array check if 'Ethiopia' exists in the array if it exists print 'ETHIOPIA'. If it does not exist add to the countries list.
```

Comment:
@Fakhara77 please reread the question do what it says
You are not doing what the question asks you to do.

Reply:
resolved and i see my mistake